### PR TITLE
Fill out wiring except for fetchmovie and a little bit of fetchenrichedmovie

### PR DESCRIPTION
**Pull request by wjlow**
My attempt at https://github.com/realestate-com-au/applied-scala/issues/11

In `fetchenrichedmovie`, we're still gonna work on `AppRoutes`. That's the bit where we use the query param matcher, and do an if statement to say "if `?enriched=true` then call `fetchEnrichedMovieHandler`, otherwise call `fetchMovieHandler`". 

Might work better as a showcase than an exercise, but it's a little weird to fill in.

### CODE REVIEW

File: src/main/scala/com/reagroup/appliedscala/AppRuntime.scala [hunk 1]
```diff
@@ -19,22 +19,54 @@ import org.http4s.client.Client
...
   private val pgsqlRepo = PostgresqlRepository(config.databaseConfig, contextShift)
 
+  private val http4sMetascoreRepo = new Http4sMetascoreRepository(httpClient, config.omdbApiKey)
+
   /**
-    * This is where we instantiate our `Service` and `Controller` for each endpoint.
-    * We will need to write a similar block for each endpoint we write.
-    */
+   * This is where we instantiate our `Service` and `Controller` for each endpoint.
+   * We will need to write a similar block for each endpoint we write.
```
Comment (millyrowboat) [inline src/main/scala/com/reagroup/appliedscala/AppRuntime.scala:30]: Should this comment change to say "You will fill out the first one and we've done the others for you"?

**Review by millyrowboat — COMMENTED** [review]
I think you'll have to fix the AppRoutesSpec files too for this; I think they had some non implemented stuff?

File: src/main/scala/com/reagroup/appliedscala/AppRuntime.scala (hunk 1 above)
Comment (wjlow) [reply src/main/scala/com/reagroup/appliedscala/AppRuntime.scala]: Done, thanks!

**Comment by wjlow** [comment]
I did update `AppRoutesSpec`. We never have to go back to it to uncomment lines of code anymore. At its current state the test for `fetchMovieHandler` will fail because it hasn't been worked on yet.

**Review by millyrowboat — APPROVED** [review]

**Pull request merged**